docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
     build:
       context: .
       args:
-        INSTALL_COMMUNITY_DEPS: false
+        INSTALL_COMMUNITY_DEPS: true
```

Comment:
Just double-checking that we want to change this

Reply:
Good catch